tests/swe_pheno.phpt

```diff
@@ -58,7 +58,7 @@ array(6) {
   [3]=>
   float(0.519086)
   [4]=>
-  float(-3.987935)
+  float(3.706732)
```

Comment:
I know this method was one of the main changes in the SE 2.10.3 release, but I still want to make sure the inversion from negative to positive is correct here?